sh/helm-template/helm-template

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
```

Comment:
1. This file needs a .sh extension
2. Directory name already qualifies this script, so rename it to something else like `run.sh`.
3. Rename top-level `sh` dir to `functions`

Reply:
Done